### ISSUE TITLE
Video story with text overlay causes the right click download action to download overlay only

### ISSUE THREAD

**Issue by IsaacHatton** [dclstn/better-snapchat#225]
See https://discord.com/channels/1217438562107002940/1217438562107002943/1225525610886070393

From 渦巻ハルト on Discord:
> there's actually something you might need to implement in order to prevent the webp text/sticker overlay from making it impossible to click on the video 
> if there are stickers or text in the video you can only save the webp overlay and not the actual video

as an simple proof of concept solution (not sure which one of these two changes made it work) if you move the img event before the video in devtools and give the image pointer-events="none" attribute it seems to direct right click events to the video
![image](https://github.com/dclstn/better-snapchat/assets/74068072/e892b24b-0b1d-493f-ad2b-799d027a7ea4)

**Comment by dclstn** [comment]
```css
.saveImage div[role='img']:has(video, img) > img {
  pointer-events: none !important;
}
```

I wonder if something like this would work

**Comment by dclstn** [comment]
Fixed with 2778d7f50c478487b66e2bb05ec043e445d9ec96 & 8ae6919bf945197ad39b4dee5a74b19a019f1bc0
